plugins/techdocs-node/src/stages/generate/helpers.ts

```diff
@@ -94,8 +94,40 @@ export const runCommand = async ({
 export const getRepoUrlFromLocationAnnotation = (
   parsedLocationAnnotation: ParsedLocationAnnotation,
   scmIntegrations: ScmIntegrationRegistry,
+): string | undefined => {
+  const { type: locationType, target } = parsedLocationAnnotation;
+
+  if (locationType === 'url') {
+    const integration = scmIntegrations.byUrl(target);
+
+    // We only support it for github, gitlab, bitbucketServer and harness for now as the edit_uri
+    // is not properly supported for others yet.
+    if (
+      integration &&
+      ['github', 'gitlab', 'bitbucketServer', 'harness'].includes(
```

Comment:
Can we move this into a constant now that we have two places using this list? Just makes it a bit easier to maintain.

Reply:
Yes good call, just made that update in the most recent commit 4f3fa75c0119c4336ea512b8d498d443e29aca38